sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -177,29 +177,53 @@
 )
 
 
-class AzureMLResourceType(object):
+class AzureMLResourceType:
```

Comment:
Is this in our public doc? This is a private namespace.

Reply:
All the objects(private/public) should be documented. We will be adding the gates to check for the docstring for classes/modules/methods, irrespective of public or private accessibility. This PR will serve as a sample PR to partner teams.